src/Orc.Controls/Orc.Controls.Shared/Controls/DatePicker/Views/DatePickerControl.xaml

```diff
@@ -40,7 +40,7 @@
                     <TextBlock Grid.Column="1" x:Name="Separator1" Text="/" Style="{DynamicResource RatioSignStyle}" />
                     <ToggleButton Grid.Column="1" x:Name="ToggleButtonD" Tag="{x:Static controls:DateTimePart.Day}"
                                   Style="{DynamicResource DatePickerLetterToggleButtonStyle}"
-                                  Checked="ToggleButton_OnChecked" Content="d"
+                                  Checked="ToggleButton_OnChecked" Content="{catel:LanguageBinding Controls_DatePickerControl_ToggleButton_Content_d}"
```

Comment:
Instead of d, use DayAbbreviation (d represents the day, but in another language it's probably not d).